### ISSUE TITLE
Bug in parallel FSI solution

### ISSUE THREAD

**Issue by hlkline** [su2code/SU2#313]
As identified by @LaSerpe during conversation in pull request https://github.com/su2code/SU2/pull/312, there is a bug in the develop branch somewhere during the merge of the solution for the parallel fsi2d test case, which may be related to recent changes in SU2_SOL. 

Error output can be seen in: https://travis-ci.org/su2code/SU2/jobs/165118367

Fix should include the addition or modification of a regression test case to catch similar errors in the future. My suggestion for where to start: go back in the builds until you get to one that does not have this error, and check out what the difference in the SU2_SOL routines are there.

**Comment by arubino** [comment]
Hi Heather, I created a branch fix_parallelFSIsol to address this issue. I had a very quick look at it and it seems to be fine now. Let me know what you think about it.
https://travis-ci.org/su2code/SU2/jobs/166032087

I can have, however, a better look next week.

I agree, we need something to test similar errors but also issues related to output files. 

PS
 @rsanfer in the develop, from last merge, in SU2_SOL there is the GetnZone method from CConfig. Can that be useful for the multizone FSI as well?

**Comment by hlkline** [comment]
Looks good, thanks for looking into this @arubino. 

I think that it's fine to start a pull request with this branch if you've made the changes you wanted - it'll fix the regression test for now, and until we find a way to test for similar errors we can try to remember to skim through log files even if there's a friendly green check-mark.

**Comment by arubino** [comment]
Hi @hlkline , shell we open a separate issue concerning just the test for similar errors?

**Comment by hlkline** [comment]
Agreed.
Closing since addressed by https://github.com/su2code/SU2/pull/315 
Thanks @arubino !
